Training 441 Crawl/css/styles.css

```diff
@@ -0,0 +1,84 @@
+/*!
+ * All Rights Reserved
+ * This software is proprietary information of
+ * Intelligent Sense
+ * Use is subject to license terms.
+ * Filename: style.css
+ */
+/*
+ * Author:      fquiros@akurey.com
+ * Date:        30/07/2019
+ * Description: Styles for training 441.
+ */
+
+
+@import url('https://fonts.googleapis.com/css?family=Acme'); 
+
+
+
+.main-container{
+    text-align: center;  
+    margin: 0 auto;
+}
+
+.title{
+    text-align: center;  
+    
+}
+.box{
+    color: white;
+    font-size: 16px;
+    text-align: left;
+    display: inline-block;
+    width: 15em;
```

Comment:
do not use em or rem for width, height, margin, padding... Use percentages or pixels instead. Only use em or rem (and try to avoid rem cause might create responsive layout issues in the future) for font-size. But if the page should not be responsive use pixels. 

As a further comment, if you use em or rem. Add a font-size in the body selector using pixels to be a base of em and rem